pupa/settings.py

```diff
@@ -9,6 +9,10 @@
 
 # scrape settings
 
+ENABLE_KAFKA = os.environ.get('ENABLE_KAFKA', "False").lower() == "true"
+KAFKA_REPORT_TOPIC = 'post-scrape-reports'
```

Comment:
Wait, a stream per jurisdiction? We have a `jurisdiction` key on the object that comes through, you can filter on that